src/main/java/com/liferay/faces/util/product/internal/ProductBaseImpl.java

```diff
@@ -39,7 +39,7 @@
 	protected int patchVersion;
 	protected String stringValue;
 	protected String title;
-	protected String version;
+	protected String version = majorVersion + "." + minorVersion + "." + patchVersion;
```

Comment:
yep, that's point. The default version will be `"0.0.0"` if the `Porduct` is not detected. However `initVersionInfo()` will set the version if the product is detected.